src/databricks/labs/ucx/hive_metastore/mapping.py

```diff
@@ -46,12 +59,20 @@ def as_uc_table_key(self):
     def as_hms_table_key(self):
         return f"hive_metastore.{self.src_schema}.{self.src_table}"
 
+    def sql_alter_to_acl_upgraded(self) -> str:
```

Comment:
Move it to the dataclass tabletomigrate

Reply:
took it out, will introduce in the PR for table acl migration